AirCasting/CreateSessionViews/ChooseSessionType/ChooseSessionTypeView.swift

```diff
@@ -220,6 +235,12 @@ struct ChooseSessionTypeView: View {
         }
     }
     
+    var sdSyncButton: some View {
+        Button("Sync data from AirBeam3") {
+            startSync = true
+        }
+    }
+    
```

Comment:
Strings and maybe one-liner?

Reply:
I'm leaving that for the sync views task